contracts/CrossChain.sol

```diff
@@ -48,6 +77,16 @@ contract CrossChain is System, ICrossChain, IParamSubscriber{
   event enableOrDisableChannel(uint8 indexed channelId, bool isEnable);
   event addChannel(uint8 indexed channelId, address indexed contractAddr);
 
+  // BEP-171: Security Enhancement for Cross-Chain Module
+  event SubmitEmergencyProposal(bytes32 indexed prosalNameHash, address indexed proposer, uint128 approveThreshold, uint128 expiredAt);
+  event EmergencySuspend(address indexed excutor);
+  event Reopen(address indexed excutor);
+  event Challenge(
+    address indexed challenger,
+    uint64 indexed packageSequence,
```

Comment:
No need to index `packageSequence` and `channelId`.

Reply:
fixed